modules/cluster-samples-operator.adoc

```diff
@@ -8,44 +8,25 @@
 [discrete]
 == Purpose
 
-The Cluster Samples Operator manages the sample imagestreams and templates
-stored in the `openshift` namespace, and any container credentials, stored as a
-secret, needed for the imagestreams to import the images they reference.
-
-On initial start up, the Operator creates the default samples resource to
-initiate the creation of the imagestreams and templates. The imagestreams are
-the {op-system-first}-based {product-title} imagestreams pointing to images on
-`registry.redhat.io`. Similarly, the templates are those categorized as
-{product-title} templates.
-
-The Cluster Samples Operator, along with its configuration resources, are
-contained within the `openshift-cluster-samples-operator` namespace. On start
-up, it will copy the pull secret captured by the installation into the
-`openshift` namespace with the name `samples-registry-credentials` to facilitate
-imagestream imports. An administrator can create any additional secrets in the
-`openshift` namespace as needed. Those secrets contain the content of a
-container `config.json` needed to facilitate image import.
-
-The image for the Cluster Samples Operator contains imagestream and template
-definitions for the associated {product-title} release. Each sample includes an
-annotation that denotes the {product-title} version that it is compatible with.
-The Operator uses this annotation to ensure that each sample matches it's
-release version. Samples outside of its inventory are ignored, as are skipped
-samples. Modifications to any samples that are managed by the Operator are
-reverted automatically. The jenkins images are part of the image payload from
-the installation and are tagged into the imagestreams directly.
-
-The samples resource includes a finalizer, which cleans up the following upon
-its deletion:
+The Cluster Samples Operator manages the sample imagestreams and templates stored in the `openshift` namespace, and any container credentials, stored as a secret, needed for the imagestreams to import the images they reference.
```

Comment:
missed this earlier ... we do not manage credentials.

Change this to 

`The Cluster Samples Operator manages the sample imagestreams and templates stored in the `openshift` namespace`

Reply:
We then need to revisit the section on samplesRegistry override and make sure we remind them that any auth credentials need to be stored as secrets in the `openshift` namespace, as well as point them to the internal registry doc for injecting certs for the new external registries.

Reply:
Looking at https://docs.openshift.com/container-platform/4.5/openshift_images/samples-operator-alt-registry.html#installation-adding-registry-pull-secret_samples-operator-alt-registry I think we are good.